src/com/library/backlog/Library.java

```diff
@@ -90,6 +94,87 @@ public void search(){
 				}
 			}
 
-		}//end of search
+		}//end of search ///////////////better would be if I use exception here book not found
+		
+		
+		
+		public void login(String inputUsername, String inputPassword){
+			
+			User userLogin = new User(User.getRole(), inputUsername, inputPassword);
+			
+			User adminstrativeUser = new User(UserRole.ADMIN, "Tayyab", "adminpass");			
+			
+			if(userList.contains(userLogin) || userList.contains(adminstrativeUser)){
+				for (User users: userList){
+					if(inputUsername.equals(users.getUsername()) && inputPassword.equals(users.getPassword())){
+						if(users.getRole().equals(UserRole.ADMIN)){
+							
+							System.out.println("You have successfully logged in as a admin and you can perform the forlowing tasks");
+							String input;
+						do{
+							
+	
+							Scanner sc = new Scanner(System.in);
+							System.out.println("To add new user enter: Add User");
+							System.out.println("To add new book enter: add book");
+							System.out.println("To quit your session enter: quit");
+							input = sc.nextLine();
+							sc.close();
+
+							if(input.equalsIgnoreCase("Add User")){
+								
+								Scanner read = new Scanner(System.in);
+								System.out.println("Please enter user name: ");
+								String usernameEntered = read.nextLine();
+								System.out.println("Please enter password: ");
+								String passwordEntered = read.nextLine();
+								read.close();
+								
+								User registerUser = new User(UserRole.USER, usernameEntered, passwordEntered);
+								registerUser.addNewUser(userList);
+							}
+							else if(input.equalsIgnoreCase("add book")){
+
+								
+								Book registerBook = addNewBook();
+								
+								
+							}
+							
+							
+						}while(input == "quit");
+						//do something 
```

Comment:
maybe take that one step further and extract the commands like start, '''quit etc into constants like private static QUIT = "quit" '''